target/linux/ath79/mikrotik/base-files/etc/board.d/02_network

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+. /lib/functions/system.sh
+. /lib/functions/uci-defaults.sh
+
+ath79_setup_interfaces()
+{
+	local board="$1"
+
+	case "$board" in
+	mikrotik,routerboard-922uags-5hpacd|\
+	mikrotik,routerboard-wap-g-5hact2hnd)
+		ucidef_set_interface_lan "eth0"
+		;;
+	*)
+		ucidef_set_interfaces_lan_wan "eth0" "eth1"
+		;;
+	esac
+}
+
```

Comment:
I'd actually add an empty ath79_setup_macs block here for convenience:

```
ath79_setup_macs()
{
	local board="$1"

	case "$board" in
	esac

	[ -n "$lan_mac" ] && ucidef_set_interface_macaddr "lan" $lan_mac
	[ -n "$wan_mac" ] && ucidef_set_interface_macaddr "wan" $wan_mac
	[ -n "$label_mac" ] && ucidef_set_label_macaddr $label_mac
}
```

Reply:
Fine, adding it.